src/core/Note.cpp

```diff
@@ -230,3 +230,8 @@ bool Note::hasDetuningInfo() const
 
 
 
+bool Note::isBetween(int tickStart, int tickEnd) const
```

Comment:
I think the name is fine.  `withinRange(int tickStart, int tickEnd) { ... }` may be another possibility. :+1: